src/dal/topic/topic.service.ts

```diff
@@ -37,20 +37,28 @@ export class TopicService {
     icon: string,
     category: Category,
   ): Promise<Topic> {
-    const topic = new Topic(name, icon, category);
-    return await this.topicRepository.save(topic);
+    const newTopic = new Topic(name, icon, category);
+    await this.topicRepository
+      .save(newTopic)
+      .then(topic =>
+        console.log('topic with name: ' + topic.name + ' created successfuly'),
+      )
+      .catch((error: Error) => console.log(error.message));
+    return await this.getTopic(newTopic.name);
   }
 
   async getTopicsByStatus(status: ApprovalStatus): Promise<Topic[]> {
     return await this.topicRepository
-      .createQueryBuilder()
+      .createQueryBuilder('topic')
```

Comment:
you can use this.topicRepository.find({ relations: ['tutorials'] 
it will have the same outcome